pyproject.toml

```diff
@@ -2,7 +2,7 @@
 requires = [
   "setuptools_scm[toml]",
   "wheel",
-  "numpy"
+  "numpy>=2.0.0"
```

Comment:
Will it fail on numpy1? 

Should it be pinned `>=2` in the pip install statements above also

Reply:
Will remove pinning. Turns out that downgrading numpy when installing and testing the wheel was not a problem.